.helm/releases/envoy-dqa.yaml

```diff
@@ -0,0 +1,81 @@
+files:
+  envoy.yaml: |-
+    static_resources:
+      listeners:
+      - name: listener_0
+        address:
+          socket_address: { address: 0.0.0.0, port_value: 10000 }
+        filter_chains:
+        - filters:
+          - name: envoy.http_connection_manager
+            config:
+              stat_prefix: ingress_http
+              use_remote_address: false
+              xff_num_trusted_hops: 1
+              route_config:
+                name: local_route
+                virtual_hosts:
+                - name: local_service
+                  domains: ["*"]
+                  rate_limits:
+                  - actions:
+                    - remote_address: {}
+                  - actions:
+                    - request_headers:
+                        header_name: :authority
+                        descriptor_key: authority
+                  - actions:
+                    - request_headers:
+                        header_name: :method
+                        descriptor_key: method
+                  - actions:
+                    - request_headers:
+                        header_name: :path
+                        descriptor_key: path
+                  - actions:
+                    - request_headers:
+                        header_name: x-forwarded-for
+                        descriptor_key: header.x-forwarded-for
+                  routes:
+                  - match:
+                      prefix: "/"
+                    route:
+                      host_rewrite: www.google.com
```

Comment:
I believe DQAs for other application would want to overwrite this `host_rewrite` value, along with some other values that reference to `google.com`, in order to have traffic proxied to the correct upstream. 

In which case, it might be helpful to make this a template file (e.g. a helm chart template file) so that people can override these values as needed. 

This could likely be done in a follow up PR though, so feel free to disregard this comment for now.

Reply:
I agree 100% that this should be configurable in the future. For Guardian-only environments though, it's useful to have google.com as the upstream so we don't need any additional upstream service present just for rate limit testing.